curious/graph.py

```diff
@@ -132,19 +132,19 @@ def get_related_objects(instances, filters=None):
 
       table = instances[0]._meta.db_table
       pk_field = instances[0]._meta.pk.column
-      related_table = field.related_field.model._meta.db_table
+      related_table = field.rel.model._meta.db_table
       if table == related_table:
         # XXX hack: assuming django uses T2 for joining two tables of same name
         table = 'T2'
       queryset = queryset.extra(select={INPUT_ATTR_PREFIX: '%s.%s' % (table, pk_field)})
 
     # reverse FK from instance to related objects with FK to the instance
-    elif type(rel_obj_descriptor) == ForeignRelatedObjectsDescriptor:
-      rel_field = rel_obj_descriptor.related.field
+    elif type(rel_obj_descriptor) == ReverseManyToOneDescriptor:
```

Comment:
isinstance?

Reply:
isinstance checks subclasses. Because of inheritance we use type